posts/inteview-with-clever-tykes.md

```diff
@@ -0,0 +1,65 @@
+
+Within the storybook there are characters from a range of backgrounds with very different personalities. We hope that we can help reaffirm the belief that difference is something to celebrate. The attendees of Code Club are all very supportive of each other's efforts!
+
+**I love that through the characters, readers are encouraged to warm to children that might be different from them in some way and from people they might be in contact with day-to-day. Let's get into the business side now, how are you spreading the word about your books?**
+
+After we had finalised the storybooks, website and distribution, and ecommerce sales were at a good level, we set about finding sponsors for the books. We are well aware that the kids who the stories benefit the most aren't the ones whose parents are able to buy them books. The first sponsorship we secured was from a technology publishing company based in Birmingham, called Packt Publishing. Packt sponsored 300 copies of Code-it Cody and gave them out to parents across the city.
+
+After that, we set our sights on gifting the books to every primary school in the UK, all 24,000 of them! We managed to secure sponsorship with Lloyds Banking Group to do exactly this in January 2017. Every primary school received a set of the Clever Tykes storybooks along with access to an online portal with resources and teaching materials for enterprise education.
+
+At the moment we're working on replicating this in the USA.
+
+**We'll certainly be keeping our eye on this! It's seems to be the dream of a lot of people to write books, especially children's storybooks, do you have any advice for readers who are considering putting pen to paper?**
+
+The main thing is just to get started! Do some research to see what else is out there on a similar topic to get an idea of where your story might fit in, then get planning and writing. Don't underestimate the value of having a great editor and illustrator and don't underestimate the challenge of marketing your book once it's produced.
+
+There are so many things to consider when planning, writing, producing and selling children's books. We are asked about this a lot, so we put everything we know in an article [here](https://medium.com/@cookiewhirls/how-to-sell-thousands-of-childrens-storybooks-without-a-publisher-898ff9544ef9).
+
+**Where can we find the Clever Tykes books?**
+
+They are available for the USA [here](http://clevertykes.com/amazonusa), the UK and Europe [here](https://amazon.co.uk/clevertykes), Japan [here](https://www.amazon.co.jp/dp/0992691389), and the rest of the world [here](http://clevertykes.com/).
```

Comment:
```suggestion
They are available for the USA at http://clevertykes.com/amazonusa, the UK and Europe here https://amazon.co.uk/clevertykes, Japan at https://www.amazon.co.jp/dp/0992691389, and the rest of the world here http://clevertykes.com/.
```

Reply:
I'm going to say no to this one to make the sentence more readable, if that's ok?

Also - as a general note, nothing in this repo currently autolinks. I need to fix this at some point.

Reply:
I see.  👍 then.